NLibrary/Classes/PlaylistMVVM/PlaylistItemBasicModel.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistBasicModel {
+    public var playlistItems: [PlaylistItemBasicModel] = []
+    public var authorId: String?
+    init?(dict: NSDictionary) {
+        guard let keyValues = dict.allValues.first as? NSDictionary else {
+            return
+        }
+        guard let authorId = dict.allKeys.first as? String else {
+            return
+        }
+        self.authorId = authorId
+        for (key, value) in keyValues {
+            guard let key = key as? String,
+                let page = value as? NSDictionary else {
+                return
+            }
+            playlistItems.append(PlaylistItemBasicModel(albumName: key, dictionary: page))
+        }
+    }
+    
+    var dict: [String : Any]?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5409525)